### PR TITLE
Cut off the Assignment name at 2 lines and use ellipsis on all assess…

### PR DESCRIPTION
**Pull request by JayArora1997**
…ments widget & margin fix

Testing instruction:
1. npm install
2. Remove (!this._isWhitelisted(ctx.url)) in the  (if (!ctx.url || !this._isWhitelisted(ctx.url)))   (d2l-upcoming-assessments-ui\node_modules\d2l-fetch-siren-entity-behavior\d2l-fetch-siren-entity-behavior.js)  (This file will be changed separately in another pr but we can remove the code for the scope of this PR)
3. npx polymer serve

https://rally1.rallydev.com/#/378960033600d/custom/382879628416?detail=%2Fuserstory%2F516992802980&fdp=true?fdp=true

Expected Outcome:
image

### CODE REVIEW

**Review by jstefaniuk-d2l — APPROVED** [review]
Code LGTM, next step here is to move the rally card to the "awaiting test" column on the board

File: components/d2l-all-assessments-list-item.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-all-assessments-list-item">
 
 			.assessment-info {
 				display: flex;
-				margin: 10px auto;
+				margin: 10px 10px;
```
Comment (adamsk34) [inline components/d2l-all-assessments-list-item.js:23]: This would be the same just slightly shorter

```suggestion
				margin: 10px;
```

**Review by adamsk34 — APPROVED** [review]

File: components/d2l-all-assessments-list-item.js (hunk 1 above)
Comment (JayArora1997) [reply components/d2l-all-assessments-list-item.js]: Yes, that sounds good. Changes have been made.

**Comment by jadasc** [comment]
Ran the Polymer server and tested resizing the window. ✔️ Meets acceptance criteria

**Pull request merged**